.github/workflows/smoke-test-yarn-berry.yml

```diff
@@ -5,16 +5,16 @@ jobs:
   chromatic:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-node@v1
-        with:
-          node-version: '16.x'
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
+      - uses: actions/setup-node@v3
+        with:
+            node-version: '16'
       - run: yarn install
       - run: yarn set version berry
       - name: run chromatic
-        run: npx -p . chromatic --build-script-name build-test-storybook --exit-zero-on-changes --force-rebuild
+        run: npx -p . chromatic --build-script-name build-test-storybook --exit-zero-on-changes
```

Comment:
Why did you need to remove `force-rebuild`? Won't this be required to rerun on the same commit?

Reply:
I don't love the idea of `--force-rebuild` being in a script all the time, but I agree that if we decide to change it, this should be outside the scope of this PR.

Reply:
@tevanoff I reverted the change 🙏🏼